extensions/ql-vscode/src/query-history.ts

```diff
@@ -736,6 +750,46 @@ export class QueryHistoryManager extends DisposableObject {
       }
     }
   }
+  
+  private warnNoEvalLog() {
+    void showAndLogWarningMessage('No evaluator log is available for this run. Perhaps it failed before evaluation, or you are running with a version of CodeQL before ' + CliVersionConstraint.CLI_VERSION_WITH_PER_QUERY_EVAL_LOG + '?');
+  }
+
+  async handleShowEvalLog(
+    singleItem: QueryHistoryInfo,
+    multiSelect: QueryHistoryInfo[]
+  ) {
+    // Local queries only
+    if (!this.assertSingleQuery(multiSelect) || singleItem?.t !== 'local') {
```

Comment:
Best if you call `const { finalSingleItem, finalMultiSelect } = this.determineSelection(singleItem, multiSelect);` before this so that you handle some edge cases where you right-click on an item that is not selected.

Reply:
Thanks, done.